docs/docs/Integrations/integrations-instana-traceloop.mdx

```diff
@@ -0,0 +1,106 @@
+---
+title: Traceloop
+slug: /integrations-instana-traceloop
+description: Instrument Langflow with the Traceloop SDK, and export traces and metrics to Instana using OpenTelemetry.
+---
+
+Traceloop SDK is a lightweight instrumentation toolkit designed for LLM applications.
+It enables developers to automatically capture and export traces, metrics, and key observability signals from their LLM-powered workflows.
+
+When combined with Instana, the exported telemetry data from Traceloop provides end-to-end visibility, allowing users to visualize traces, analyze performance bottlenecks, and ensure reliable operation of LLM-driven applications.
+
+This guide demonstrates how to integrate the Instana observability platform with your Langflow application using the Traceloop SDK so you can monitor and analyze LLM performance.
+
+## Prerequisites
+
+- Create a [Traceloop API key](https://app.traceloop.com/settings/api-key)
+- Create an [Instana endpoint and Instana key](https://www.ibm.com/docs/en/instana-observability/1.0.302)
+- [Install Langflow](/get-started-installation)
+
+## Configure environment variables
+
+1. In the root folder of your Langflow application, edit your existing Langflow `.env` file or create a new one.
+
+2. Enter the following environment variables, and then replace the placeholders with the values for your deployment or requirements:
+
+   ```text
+   TRACELOOP_API_KEY=tl_dummy_1234567890abcdef1234567890abcdef
+   TRACELOOP_BASE_URL=https://otlp-magenta-saas.instana.rocks:4318
+   TRACELOOP_HEADERS="x-instana-key=INSTANA_KEY"
+   OTEL_EXPORTER_OTLP_INSECURE=false
+   TRACELOOP_METRICS_ENDPOINT=HOST:8000
+   TRACELOOP_METRICS_ENABLED=true
+   OTEL_METRIC_EXPORT_INTERVAL=10000
+   ```
```

Comment:
Do we need to add these to the global variables page, @mendonk ?https://docs.langflow.org/configuration-global-variables#default-environment-variables